MobileCenter/MobileCenter/Internals/Utils/MSUtility+File.m

```diff
@@ -0,0 +1,78 @@
+#import "MSLogger.h"
+#import "MSMobileCenterInternal.h"
+#import "MSUtility+File.h"
+
+/*
+ * Workaround for exporting symbols from category object files.
+ */
+NSString *MSUtilityFileCategory;
+
+@implementation MSUtility (File)
+
++ (BOOL)createFileAtURL:(NSURL *)fileURL {
+  if (fileURL) {
+
+    // Check if file already exists.
+    if ([fileURL checkResourceIsReachableAndReturnError:nil]) {
+      return YES;
+    }
+
+    // Create parent directories as needed.
+    NSURL *directoryURL = [fileURL URLByDeletingLastPathComponent];
+    if (![directoryURL checkResourceIsReachableAndReturnError:nil]) {
+      [self createDirectoryAtURL:directoryURL];
+    }
+
+    // Create the file.
+    if ([[NSData data] writeToURL:fileURL atomically:NO]) {
+      return YES;
+    } else {
+      MSLogError([MSMobileCenter logTag], @"Couldn't create new file at path %@", fileURL);
+    }
+  }
+  return NO;
+}
+
++ (BOOL)createDirectoryAtURL:(NSURL *)directoryURL {
+  if (directoryURL) {
+
+    // Create directory also create parent directories if they don't exist.
+    NSError *error = nil;
+    if ([[NSFileManager defaultManager] createDirectoryAtURL:directoryURL
+                                 withIntermediateDirectories:YES
+                                                  attributes:nil
+                                                       error:&error]) {
+      [self disableBackupForDirectoryURL:directoryURL];
+      return YES;
+    } else {
+      MSLogError([MSMobileCenter logTag], @"Couldn't create directory at path %@: %@", directoryURL,
+                 error.localizedDescription);
+    }
+  }
+  return NO;
+}
+
++ (BOOL)removeItemAtURL:(NSURL *)itemURL {
+  NSError *error = NULL;
```

Comment:
I think it doesn't matter since it's an ObjC pointer, I probably copy past it from somewhere, I'll harmonize it.